ts/components/VirtualTable.svelte

```diff
@@ -0,0 +1,80 @@
+<!--
+Copyright: Ankitects Pty Ltd and contributors
+License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
+-->
+<script lang="ts">
+    let className: string = "";
+    export { className as class };
+
+    export let itemsCount: number = 0;
+    export let containerHeight: number;
+    export let itemHeight: number;
+
+    let container: HTMLElement;
+    let scrollTop: number = 0;
+
+    $: innerHeight = Math.max(containerHeight, itemsCount * itemHeight);
+    $: sliceLength = Math.ceil(containerHeight / itemHeight);
+    $: startIndex = Math.floor(scrollTop / itemHeight);
+    $: endIndex = Math.min(startIndex + sliceLength, itemsCount);
+    $: slice = new Array(endIndex - startIndex).fill(0).map((_, i) => startIndex + i);
```

Comment:
While scrolling the table quickly, I see this appear from time to time:

JS error /_anki/pages/import-log.js:1 Uncaught TypeError: Cannot convert undefined or null to object

Not sure what's causing it.

Reply:
It's a Bootstrap issue: https://github.com/twbs/bootstrap/issues/37474

I pushed a fix.

Reply:
Nice, thanks for getting to the bottom of that.